samples/encryption.js

```diff
@@ -122,13 +122,34 @@ function downloadEncryptedFile(bucketName, srcFilename, destFilename, key) {
   // [END storage_download_encrypted_file]
 }
 
-// [START storage_rotate_encryption_key]
-function rotateEncryptionKey() {
-  throw new Error(
-    'This is currently not available using the Cloud Client Library.'
-  );
+function rotateEncryptionKey(bucketName, fileName, oldKey, newKey) {
+  // [START storage_rotate_encryption_key]
+  // Imports the Google Cloud client library
+  const Storage = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const fileName = 'Nome of a file in the bucket, e.g. my-file';
```

Comment:
I think oldKey and newKey should be apart of this const list of variables.

Reply:
I don't believe so. I've been following the format of the previous examples in this file, such as this method: https://github.com/stephenplusplus/nodejs-storage/blob/18c92a222262708b0a6a9866eaed3eea8431798b/samples/encryption.js#L89

Reply:
I feel that this is a typo given other samples such as https://github.com/stephenplusplus/nodejs-storage/blob/18c92a222262708b0a6a9866eaed3eea8431798b/samples/files.js#L57 which define a commented variable for users determined by method parameters.

Reply:
I see now, and I would have to agree!

Reply:
Added in the `const` declarations 👍